api/actions/createChangelist.js

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+const GithubEvents = require('../modules/GithubEvents');
+const Issues = require('../services/Issues');
+
+module.exports = {
+    eventName: GithubEvents.events.PULL_REQUEST_CLOSED,
+    fn: async function(req) {
+        const {
+            pull_request: { title, url, closed_at }
+        } = req.body;
+        if (title.includes('@W-')) {
```

Comment:
it seems to be easy criteria to link a GUS work item to PRs ... but there are some use cases where it might be confusing e.g. the creation of a work item in GUS could be automatically by adding the label GUS P1 into an issue ... and at that moment I don't know the @W- id that will be created ... moving forward if I want to link a PR to that @W- I will need to navigate to GUS ... find the work item and go back and paste it in the title

My point is there has to be a way for anyone working in Github to no use GUS at all because git2gus is smart an does the magic for you.

## How? 
PRs and Issues in GitHub can be linked by pasting the link in the PR. if the issue is linked with the work item then the PR could be related to the work item in GUS

makes sense? :)

Reply:
That makes sense. This will provide a way for our non-Salesforce contributors to link their PR, Issue and GUS. Will it be ok to take this up as an additional feature? I want to see the first cut works as expected and will be happy to add any other improvements.